templates/terraform/environments/prod/main.tf

```diff
@@ -110,6 +110,19 @@ module "prod" {
   sendgrid_enabled = <%if eq (index .Params `sendgridApiKey`) "" %>false<% else %>true<% end %>
   sendgrid_api_key_secret_name = "${local.project}-sendgrid-<% index .Params `randomSeed` %>"
 
+  # Cache configuration
+  ## you may define "redis" or "memcached" as your cache store. If you define "none", there will be no cache service launched.
+  ## Check https://docs.aws.amazon.com/AmazonElastiCache/latest/mem-ug/SelectEngine.html to compare redis or memcached.
+  cache_store = "<% index .Params `cacheStore` %>"
+
+  ## See how to define node and instance type: https://docs.aws.amazon.com/AmazonElastiCache/latest/mem-ug/nodes-select-size.html
+  cache_cluster_size  = 1
+  cache_instance_type = "cache.r6g.large"
+
+  ## for Redis only
+  ### when this is enabled, your application needs to handle TLS connection. Note: redis-cli can not handle TLS.
+  cache_transit_encryption_enabled = true
```

Comment:
Let's leave this out and just have it use the default. Most people won't really care about this option. Maybe move this comment into the description of the variable?